libs/wire-api/src/Wire/API/Routes/Public/Galley/Conversation.hs

```diff
@@ -342,14 +344,15 @@ type ConversationAPI =
                :> Description "This returns 201 when a new conversation is created, and 200 when the conversation already existed"
                :> ZLocalUser
                :> ZOptClient
-               :> ZConn
+               :> ZOptConn
```

Comment:
I'm sure this was justified, but can you clarify why is the connection ID now optional? Only the local user can call this endpoint, hence I believe we always have the connection ID.

CC: @fisx , @battermann

Reply:
When this endpoint is called by an OAuth application, we do not have a connection ID.